docs/sdk/serverless-signing/combining-signatures.md

```diff
@@ -46,14 +51,15 @@ const res = await client.executeJs({
     }
 });
 
-console.log("response from singing in a transaction: ", res);
+console.log("response from signing in a transaction: ", res);
 ```
 
 ## Signing a Transaction
-With the built in `EthersJS` we are able to take advantage of the `serializeTxnForSigning` implementations and serialize a transaction, which is then signed, combined and then sent back to the client.
+
+The following Lit Action uses `EthersJS` to serialize and sign a transaction (combining signature shares from the Lit nodes) before sending it back to the client where it can be broadcasted to chain. 
```

Comment:
```suggestion
The following Lit Action uses `ethers.js` to serialize and sign a transaction (combining signature shares from the Lit nodes) before sending it back to the client where it can be broadcasted to chain. 
```